themes/scss/widgets/chart.scss

```diff
@@ -92,7 +92,7 @@
 
 .CDB-Chart-axisTipText {
   @include css3-prefix(user-select, none);
-  font-size: $sFontSize-smaller;
+  font-size: $sFontSize-small;
```

Comment:
Name of variable `sFontSize-small` should be written in all lowercase letters with hyphens instead of underscores